package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "namely-ui-css",
-  "version": "0.2.1",
+  "version": "0.3.1",
```

Comment:
idk

Reply:
minor makes sense - if I understand correctly it's a new "feature", not a bug fix (patch) and not a breaking change or official release (major)